packages/blade/src/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,160 @@
+} & StyledPropsBlade &
+  TestID;
+
+const FocussableTag = styled(BaseBox)<{ isTagFocussed: TagProps['_isTagFocussed'] }>((props) => {
+  if (props.isTagFocussed) {
+    return {
+      outline: `1px solid ${props.theme.colors.surface.background.level1.lowContrast}`,
+      boxShadow: `0px 0px 0px 4px ${props.theme.colors.brand.primary[400]}`,
+    };
+  }
+
+  return {};
+});
+const Desktop = ({ children }: { children: (React.ReactElement | null)[] }): React.ReactElement => (
+  <Box display={{ base: 'none', m: 'flex' }} alignItems="center" flexDirection="row">
+    {children}
+  </Box>
+);
+
+const Mobile = ({ children }: { children: (React.ReactElement | null)[] }): React.ReactElement => (
+  <Box display={{ base: 'flex', m: 'none' }} alignItems="center" flexDirection="row">
+    {children}
+  </Box>
+);
+
+const Tag = ({
+  size = 'medium',
+  icon: Icon,
+  onDismiss,
+  children,
+  isDisabled,
+  testID,
+  _isTagFocussed,
+  ...styledProps
+}: TagProps): React.ReactElement | null => {
+  const textColor = isDisabled
+    ? 'surface.text.placeholder.lowContrast'
+    : 'surface.text.subtle.lowContrast';
+
+  const mediumPadding: PaddingProps['padding'] = {
+    base: ['spacing.2', 'spacing.3', 'spacing.2', 'spacing.4'],
+    m: ['spacing.1', 'spacing.2', 'spacing.1', 'spacing.3'],
+  };
+
+  const largePadding: PaddingProps['padding'] = {
+    base: ['spacing.2', 'spacing.3', 'spacing.2', 'spacing.4'],
+    m: ['spacing.2', 'spacing.3', 'spacing.2', 'spacing.4'],
+  };
+
+  const getLeadingIcon = ({ size }: { size: IconProps['size'] }): React.ReactElement | null =>
+    Icon ? (
+      <Box>
+        <Icon color={textColor} size={size} marginRight="spacing.2" />
+      </Box>
+    ) : null;
+
+  const getTagText = ({
+    size,
+  }: {
+    size: TextProps<{ variant: 'body' }>['size'];
+  }): React.ReactElement => (
+    <Box maxWidth={makeSize(globalSizeTokens['100'])}>
+      <Text truncateAfterLines={1} marginRight="spacing.2" color={textColor} size={size}>
+        {children}
+      </Text>
+    </Box>
+  );
+
+  const getCloseIcon = ({ size }: { size: IconButtonProps['size'] }): React.ReactElement => (
+    <IconButton
+      size={size}
+      icon={CloseIcon}
+      accessibilityLabel={`Close ${children} tag`}
+      isDisabled={isDisabled}
+      onClick={() => {
+        onDismiss();
+      }}
+    />
+  );
+
+  return (
+    <FocussableTag
+      display={(isReactNative() ? 'flex' : 'inline-flex') as never}
+      alignSelf={isReactNative() ? 'center' : undefined}
+      flexDirection="row"
+      flexWrap="nowrap"
+      backgroundColor="brand.gray.a100.lowContrast"
+      borderRadius="max"
+      padding={size === 'medium' ? mediumPadding : largePadding}
+      {...getStyledProps(styledProps)}
+      {...metaAttribute({ name: MetaConstants.Tag, testID })}
+      isTagFocussed={_isTagFocussed}
+    >
+      <Box display="flex" flexDirection="row" flexWrap="nowrap" alignItems="center">
+        <Desktop>
```

Comment:
Maybe use a better component name that's easier to understand? `<ShowOnDesktop>` & `<ShowOnMobile>` maybe?

Reply:
done